src/apps/appstore/style/app.scss

```diff
@@ -225,6 +255,20 @@
 	font-size: 12px;
 }
 
+#appstore_container .market-beta-notice {
+	margin: 0 auto 25px;
+	padding: 24px;
+
+	background-color: #c6d9fc;
+	border-radius: 2px;
+	font-size: 16px;
+
+	i.fa-info-circle {
+		margin-right: 10px;
+		color: #3478f5;
+		font-size: 18px;
+	}
+}
```

Comment:
These styles may no longer be necessary if the [monsterPanelText](https://docs.2600hz.com/ui/docs/handlebars/#monsterpaneltext) handlebars helper is used.